common/src/main/java/com/wynntils/services/mapdata/attributes/AbstractMapAttributes.java

```diff
@@ -27,8 +28,8 @@ public int getPriority() {
     }
 
     @Override
-    public int getLevel() {
-        return 0;
+    public RangedValue getLevelRange() {
```

Comment:
I think we never fully agreed on how to handle this. Does this mean "skill level is not relevant for this marker", or "the minimum level is not known/provided" or "this is available to all players"? Should we separate these concerns?